docs/source/index.html

```diff
@@ -0,0 +1,105 @@
+<div class="doc-body">
+    <section class="hero">
+        <div>
+            <h1>Take Full Control of Your Data with JAX AI Stacks</h1>
```

Comment:
This looks like an early draft of the content copy – we'll need to update before we can merge this PR.

Reply:
It is an early version, I'm still waiting on final copy but will update soon.